src/kubernetes-ingress-controller/guides/using-gateway-api.md

```diff
@@ -214,3 +231,7 @@ Gateway APIs are supported. In particular:
   configuration, and must either accept the default Kong certificate or add
```

Comment:
I think in 2.5, `TLSConfig` is supported, so here we should remove this section in docs for versions >= 2.5